Libraries/Components/Pressable/Pressable.js

```diff
@@ -210,17 +240,44 @@ function Pressable(props: Props, forwardedRef): React.Node {
   );
   const eventHandlers = usePressability(config);
 
+  const pressableTVEventHandler = (evt: Event) => {
+    if (props.isTVSelectable !== false || props.focusable !== false) {
+      const tag = findNodeHandle(viewRef.current);
+      if (tag === evt.tag) {
```

Comment:
Disclaimer: I don't know anything about the TV SDK.

If a given Pressable has received an event, why do you need to check if the tags match? Is that because you're using the useTVEventHandler hook? 
Afaik, the way event dispatching works is that a view manager includes the tag in the event payload, and RN funnels it to  the correct node, so eg. if you press a Pressable, you don't need to check the tag in the incoming event, but rather the onPress callback gets invoked accordingly. 
Which ViewManager does the focus/blur event come from here? On Windows it's the ViewViewManager, so we just hook up the onFocus stuff to the View that Pressable wraps (right below this function, in render()).

Reply:
@rectified95 your intuition is correct... the useTVEventHandler hook doesn't keep info on the component that fired the event.  This doesn't matter for remote control events (up/down/left/right/select) but it does matter for focus/blur.  Now that @TheSavior has provided me the correct APIs for 0.63, I can go and fix useTVEventHandler and make Pressable work correctly.